helm/hpcc/templates/esp.yaml

```diff
@@ -113,6 +115,13 @@ spec:
       serviceAccountName: "hpcc-default"
       containers:
       - name: {{ .name | quote }}
+{{ if hasKey . "service" }}
+ {{ if hasKey .service "servicePort" }}
```

Comment:
Why don't ESP containers report containerport? From my reading, it's mostly "informative". We find ourselves requiring a containerport on ESP pod for Prometheus pod discovery. Any concerns?